src/modal.js

```diff
@@ -233,6 +235,8 @@ export function setModalAsClosed(element) {
   devAssert(isConnectedNode(element));
   devAssert(topModalElement === element);
 
+  element.removeAttribute('aria-modal');
```

Comment:
Based on [this example](https://www.w3.org/TR/wai-aria-practices/examples/dialog-modal/dialog.html), it looks like modals that are closed do not need to remove the attribute so long as they are properly hidden (which it looks like they are in the code snippet following this one).
```suggestion
```